src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,118 @@
+package duke.command;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.Scanner;
+
+import duke.task.*;
+
+/**
+ * This class is an implementation of Duke customised to be named LOTTERY-A
+ * that stands for List of Tasks That Eventually Require Your Attention.
+ * It is a CLI designed to work as a todolist.
+ *
+ * CS2103T ip
+ * AY21/22 Semester 1
+ *
+ * @author Kishendran Vendar Kon (Group G05)
+ */
+public class Duke {
+    /** Contains task list. */
+    private TaskList tasks;
+    /** Deals with loading and saving tasks from and to file. */
+    private Storage storage;
+
+    private MainWindow mainWindow;
+
+    /** Default Constructor. */
```

Comment:
Maybe it should be three lines (instead of only one)?
```suggestion
    /** 
     *Default Constructor. 
     */
```